apps/dashboard/src/components/maily/maily-config.tsx

```diff
@@ -197,6 +194,20 @@ export const createExtensions = ({
   isTranslationEnabled?: boolean;
   translationKeys?: TranslationKey[];
   onCreateNewTranslationKey?: (translationKey: string) => Promise<void>;
+  variableSuggestionsPopover?: ForwardRefExoticComponent<{
+    items: Variable[];
+    onSelectItem: (item: Variable) => void;
+  }>;
+  renderVariable: (opts: {
+    variable: Variable;
+    fallback?: string;
+    editor: Editor;
+    from: 'content-variable' | 'bubble-variable' | 'button-variable';
+  }) => JSX.Element | null;
+  createVariableNodeView: (
+    variables: LiquidVariable[],
+    isAllowedVariable: IsAllowedVariable
+  ) => (props: NodeViewProps) => JSX.Element;
```

Comment:
These components previously had workflow-related logic, so I changed them to be pure and reused for the layouts. However, they keep that logic in the workflow editor.